src/main/java/com/auth0/AuthorizeUrl.java

```diff
@@ -51,23 +51,6 @@
                 .withScope(SCOPE_OPENID);
     }
 
-    /**
-     * Creates a new instance that can be used to build an Auth0 Authorization URL.
-     *
-     * Using this constructor will store the state and nonce in the {@link javax.servlet.http.HttpSession}, and will be
-     * deprecated and removed in the future, in favor of storing the state and nonce in Cookies.
-     *
-     * @param client       the Auth0 Authentication API client
-     * @param request      the request where the state and nonce will be stored in the {@link javax.servlet.http.HttpSession}
-     * @param redirectUrl  the url to redirect to after authentication
-     * @param responseType the response type to use
-     */
-    // TODO - deprecate in minor version, remove in next major
```

Comment:
Shouldn't this be a `@Deprecated` instead for a gradual removal like the other two?

Reply:
This constructor is package-private, so not publicly available to use. The comment to deprecate was a mistake, we can just remove.